src/main/java/sx/blah/discord/handle/impl/obj/User.java

```diff
@@ -221,6 +222,23 @@ public void setDiscriminator(String discriminator) {
 
 		return new LinkedList<>();
 	}
+	
+	@Override
+	public Color getColorForGuild(IGuild guild) {
+		Color color = new Color(0);
+		int maxPosition = -1;
+		List<IRole> roleList = getRolesForGuild(guild);
+		if (roleList.isEmpty()) return color;
+		
+		for (IRole role : roleList) {
+			if (role.getPosition() > maxPosition && !role.getColor().equals(color)) {
```

Comment:
This will break if, for example, the second highest role has a non-zero color but the highest does